mgmtsystem_security_event/models/mgmtsystem_risk_matrix.py

```diff
@@ -0,0 +1,187 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 - Present
+#    Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, orm
+from openerp.tools.translate import _
+
+
+MATRIX_TYPES = [
+    ('original', _('Before applying any measure')),
+    ('current', _('With current measures')),
+    ('residual', _('After applying the planned measures')),
+]
+
+MATRIX_TYPE_DICT = {t[0]: t[1] for t in MATRIX_TYPES}
```

Comment:
1. this can be simply `dict(MATRIX_TYPES)`
2. Why is this used? There must be a better way.